src/h5web/visualizations/heatmap/config.ts

```diff
@@ -43,7 +43,7 @@ export const [useHeatmapConfig] = createPersistableState<HeatmapConfig>(
     setCustomDomain: (customDomain: Domain | undefined) =>
       set({ customDomain }),
 
-    colorMap: 'Magma',
+    colorMap: 'Viridis',
```

Comment:
I put Viridis first in the list, so makes sense to make it the default, I think.